cryptus-app/components/wallet_viewer/profile_wallet_viewer/collection_block/collection_block_name/collection_block_name.module.scss

```diff
@@ -4,13 +4,8 @@
   display: flex;
   flex-direction: column;
   align-items: center;
-  width: 85%;
   text-align: center;
 
-  height: auto;
-  background-color: v.$bg-color;
-  color: v.$text-color;
-  margin: auto;
-
-  font-weight: bold;
+  margin-left: 7.5%;
+  margin-right: 7.5%;
```

Comment:
don't use % as margins, use px. And use combined margin syntax
`margin: 0px 8px` does 0 margin at top and bottom, and 8px at left and right
`margin: 1px 2px 3px 4px` does 1 at top, 2 at right, 3 at bottom, 4 at left